File: Code_Coach/java/Halloween_Candy.java
import java.util.Scanner;
import java.lang.*;

public class Halloween_Candy{
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int houses = input.nextInt();
        houses = (int)(Math.ceil(2.0/houses*100));
        //your code goes here
        System.out.println(houses);
    }
}

File: Code_Coach/c/Halloween_Candy.c
#include <stdio.h>
#include <math.h>
int main() {
    int houses;
    float x;
    scanf("%d", &houses);

    //your code goes here
    x = (2.0/houses)*100;
    houses = ceil(2/houses*100);
    printf("%.0f",houses,ceil(x));
    return 0;
}